db/new_seeds/base/add_contracts.rb

```diff
@@ -1,3 +1,4 @@
 # frozen_string_literal: true
 
 Importers::CreateCallOffContract.new.call
+Importers::CreateMentorCallOffContract.new.call
```

Comment:
should we add this to create new ecf contract as well?

Reply:
added